includes/rest-api.php

```diff
@@ -603,6 +686,136 @@ function pmpro_rest_api_get_checkout_levels( $request ) {
 			return new WP_REST_Response( $r );
 		}
 
+
+		/// ZAPIER TRIGGERS
+		/**
+		 * Function used for testing authentication through the Zapier API.
+		 *
+		 * @since 2.6.0
+		 */
```

Comment:
```suggestion
		/**
		 * Handle authentication testing for the Zapier API.
		 *
		 * @since 2.6.0
		 *
		 * @param WP_REST_Request $request The REST request.
		 */
```